apt/minimization/minimizer.py

```diff
@@ -227,9 +228,9 @@ def fit(self, X: Union[np.ndarray, pd.DataFrame] = None, y: Union[np.ndarray, pd
             used_data = X
             if self.train_only_QI:
                 used_data = x_QI
-            X_train, X_test, y_train, y_test = train_test_split(X, y, stratify=y,
-                                                                test_size=0.4,
-                                                                random_state=18)
+            X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.4,
```

Comment:
I'm getting a bunch of failed tests now when running this and I think it may be because you removed this stratify=y param. I would leave it (within an if statement, only for classification and not regression).

Reply:
that was because I changed the random state by mistake, but I will leave the stratify param